falcon/routing/converters.py

```diff
@@ -31,18 +31,38 @@
 class BaseConverter(metaclass=abc.ABCMeta):
     """Abstract base class for URI template field converters."""
 
+    CONSUME_PATH = False
```

Comment:
this should probably renamed in `consume_multiple_segments` or similar so that it keeps making sense if in the future we enable support for `prefix/{rest:path}/suffix`